modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/Invites/index.js

```diff
@@ -67,6 +45,31 @@ const Invites = () => {
 		}=${project.accountKey}`;
 	};
 
+	const handleSubmit = () => {
+		if (!filledEmails) {
```

Comment:
We could inverse the order of this if and else.

if (filledEmails) {
...
} else {

Reply:
Thanks, Solved!